components/ntp_background_images/browser/BUILD.gn

```diff
@@ -24,6 +24,8 @@ static_library("browser") {
 
   deps = [
     "//base",
+    "//brave/common:pref_names",
+    "//brave/common:switches",
```

Comment:
FYI, we want to delete chrome(brave) layer dependency(`//brave/browser`, `//brave/common` from `//brave/components`).
I think we don't need to add this (`//brave/common/switches`).
Can you delete `#include "brave/common/brave_switches.h"` from `components/ntp_background_images/browser/ntp_background_images_service.cc`?

Also, `//brave/common:pref_names` should not be used.
Currently, `brave/common/pref_names.h` is included because `ntp_background_images` refers prefs of  `brave_referrals_service`. They are `kReferralPromoCode`, `kReferralInitialization` and `kReferralCheckedForPromoCodeFile`. I think they should be defined in `brave_referrals_service`
If we move prefs from `//brave/common/pref_names` to `//brave//components/brave_referrals_service/common/pref_names.h`, we also don't need to add brave layer dependency here. WDYT? cc @emerick

Reply:
Agreed, that makes sense.

Reply:
@emerick thanks for checking! @Gyuyoung Can you handle this?

Reply:
@simonhong @emerick Thanks for the comments. Let me update them and request a new PR on b-c repo directly.